### ISSUE TITLE
Add support for env_file

### ISSUE THREAD

**Issue by igor-ritual** [kurtosis-tech/kurtosis#1908]
### Background & motivation

While porting services from docker compose I'm unable to port env_file:
https://docs.docker.com/compose/compose-file/05-services/#env_file

### Desired behaviour

Support env_file in ServiceConfig

### How important is this to you?

Painful; the lack of this feature makes using Kurtosis frictionful.

### What area of the product does this pertain to?

SDK: the Software Development Kit libraries - Typescript, Go, etc.

**Comment by leeederek** [comment]
Hello again @igor-ritual! Have you tried using the `env_vars` field in `ServiceConfig`? https://docs.kurtosis.com/api-reference/starlark-reference/service-config/ 

If yes, could you share more about what specific errors or issues you're hitting when porting over your env_file from your docker-compose? If you got a reference or example, that'd help a lot too!

**Comment by igor-ritual** [comment]
> Have you tried using the env_vars field in ServiceConfig
Yes, however the service is published with an env file so it is painful to keep it in sync.

**Comment by leeederek** [comment]
Gotcha! You can make those `env_vars` parameters that you or anyone using your package can pass in as arguments at runtime! Example:

In your `main.star` file, you can have a function like so:
```
def run(plan):
    your_package.run(plan, API_KEY)
```

Then, when you go to run the package, you or your users can pass in the environment variable like so: `kurtosis run main.star '{"API_KEY": "abc1234"}'` 

Does that make sense?
